.github/actions/shielder-relayer/deploy/action.yml

```diff
@@ -21,15 +21,14 @@ inputs:
     description: Kustomize version
     required: true
   environment:
+    description: Environment to deploy to
     type: choice
     options:
+      - all (dev, stage, prod)
```

Comment:
I see the intention behind the bracket, but can we make it just `all` -- I'm always scared to have whitespace in these kind of variables.

Reply:
shortened